core/src/main/java/org/apache/iceberg/rest/auth/OAuth2Properties.java

```diff
@@ -37,8 +37,8 @@ private OAuth2Properties() {
    * Interval in milliseconds to wait before attempting to exchange the configured catalog Bearer token.
    * By default, token exchange will be attempted after 1 hour.
    */
-  public static final String EXCHANGE_TOKEN_MS = "exchange-token-in-ms";
-  public static final long EXCHANGE_TOKEN_MS_DEFAULT = 3_600_000; // 1 hour
+  public static final String TOKEN_EXPIRES_IN_MS = "token-expires-in-ms";
```

Comment:
This rename brings the property more in line with the OAuth2 `expires_in` field.

Reply:
OAuth spec has this as seconds.  Should we just use the unit from the spec?  I'm not sure milliseconds add much value.

Reply:
All other durations in Iceberg config use milliseconds, so I did this for consistency. I agree it doesn't help much, but I think consistency is important so people don't always need to look up the expected unit.